### ISSUE TITLE
unittest corrected

### ISSUE THREAD

**Issue by s4il3sh** [ubsuny/23-Homework4G2#82]
I guess there were errors at first in the unittest file, right? Could you please point it out? I saw one issue in the unittest file in the documentation file. @Pranjal-Srivastava-2023 did you fix it?

**Comment by Pranjal-Srivastava-2023** [comment]
@s4il3sh the documentation has pics of when the unit test failed. It was fixed after I modified your spring constant code. The documentation also mentions what changes were made to fix the issue.

**Comment by s4il3sh** [comment]
I guess the documentation just provides slight information but not the complete picture of what has been done.

**Comment by reshnashrestha** [comment]
@s4il3sh yeah in our case also documentation covers a lot less than what has been done.
